tests/components/aqara/test_util.py

```diff
@@ -0,0 +1,17 @@
+"""Tests for the Aqara switch device."""
+# from unittest.mock import patch
+
+
+from homeassistant.components.aqara import (  # remap_value,
+    string_dot_to_underline,
+    string_underline_to_dot,
+)
+from homeassistant.core import HomeAssistant
+
+
+async def test_remap_value(hass: HomeAssistant) -> None:
+    """Tests remap_value."""
+    # print(remap_value(10, 0, 100, 0, 50, False))
+    # assert remap_value(10, 0, 100, 0, 50, True) == 20
```

Comment:
stale code

Reply:
Unresolved this comment, there is still stale code here.

Reply:
removed the file